src/main/kotlin/no/nav/syfo/melding/kafka/producer/DialogmeldingBestillingProducer.kt

```diff
@@ -22,9 +22,13 @@ class DialogmeldingBestillingProducer(
                     dialogmeldingBestillingDTO
                 )
             ).get()
+
             COUNT_KAFKA_CONSUMER_MELDING_TIL_BEHANDLER_BESTILLING_SENT.increment()
```

Comment:
Jeg beholdt denne for å opprettholde total-counteren, men trenger vi den? Vet ikke om vi trenger å vise den frem i Grafana, men kan jo være grei å ha likevel 🤷🏼‍♂️

Reply:
Alternativt kan vi jo summere de andre, men da må man huske å endre når det kommer flere typer 🤔